packages/main/src/RPA/Excel/Files.py

```diff
@@ -291,13 +295,16 @@ def open_workbook(self, path: str, data_only: bool = False):
 
         .. code-block:: robotframework
 
-            # Open workbook with only path provided
-            Open Workbook    path/to/file.xlsx
+            *** Keywords ***
+            Open workbook with only path provided
+                Open Workbook    path/to/file.xlsx
 
-            # Open workbook with path provided and reading formulas in cells
-            # as the value stored
-            # Note: Can only be used with XLSX workbooks
-            Open Workbook    path/to/file.xlsx    data_only=True
+            *** Keywords ***
+            Open workbook with path provided and data only
+                [Documentation]    This example can only be used with XLSX workbooks
+                ...    and will read the cells as values, even if formulas are in
+                ...    the cell
+                Open Workbook    path/to/file.xlsx    data_only=True
```

Comment:
```suggestion
                Open Workbook    path/to/file.xlsx    data_only=${True}
```

Even if it has the same behaviour, `True` is the string `"True"` while `${True}` (even all-caps) is the boolean. (as `False` will be **true** while `${False}` is **false**)